hc-sr04-range-sensor/Makefile

```diff
@@ -61,6 +64,18 @@ OUT := out
 ELF0 := $(OUT)/pru-halt.elf
 ELF1 := $(OUT)/hc-sr04-range-sensor.elf
 
+#Installation, start stop related variable
+ARCHT   :=$(shell uname -m)
```

Comment:
Macros and $(shell) invocations should be avoided. Especially if you are writing example code. See below for suggestion how to remove this line.